install.sh

```diff
@@ -16,31 +16,20 @@
 
```

Comment:
I've been testing the re-running since December, it should work.  That's why we disable the RIDS service before updating and re-enabling it, and why source was freshly cloned.

With pip it is a little different because pip will reuse the cached repo files and the eggs of dependencies, but it should fetch the new version and install if the branch name changes (though, it looks like it may check the version field of project and only upgrade if that changes).

I've tried to keep it re-entrant, even with this latest change to add a non-root user for running pip.

Reply:
I haven't tested the latest version in a droplet yet, I've lost access to the one I had.